site/deploy-site.js

```diff
@@ -0,0 +1,46 @@
+const { execSync } = require('child_process');
+const process = require('process');
+
+const execWithOutput = (cmd) => execSync(cmd, { stdio: 'inherit' });
+
+const DEPLOY_REPOSITORY = 'https://github.com/embark-framework/embark-site';
+const DEPLOY_ORIGIN = 'embark-site';
+
+const TMP_DEPLOY_BRANCH = 'embark-site-deploy';
+const LOCAL_DEPLOY_BRANCH = 'embark-site-deploy-dist';
+const REMOTE_DEPLOY_BRANCH = 'gh-pages';
+const MASTER_BRANCH = 'master';
+
+const SITE_DIR = 'site';
+const PUBLIC_DIR = 'public';
+const ROOT_DIR = process.cwd();
```

Comment:
```suggestion
const ROOT_DIR = path.join(__dirname, '..');
```

Could be a more reliable way to calculate the root, so that it's correct regardless of the directory from which the script was invoked.

Reply:
Ah thanks!